examples/freertos/Main.cpp

```diff
@@ -0,0 +1,68 @@
+#include "generated/stm32fxxx/PinoutTableDefault.hpp"
+#include "hal_st/instantiations/NucleoUi.hpp"
+#include "hal_st/instantiations/StmEventInfrastructure.hpp"
+#include "osal/Osal.hpp"
+#include "osal/freertos/LowPowerStrategyFreeRtos.hpp"
+#include "osal/freertos_system_time/TimerServiceFreeRtos.hpp"
+#include "services/util/DebugLed.hpp"
+#include <chrono>
+#include <thread>
+
+extern "C" void Default_Handler()
+{
+    hal::InterruptTable::Instance().Invoke(hal::ActiveInterrupt());
+}
+
+main_::NucleoF767ziUi ui;
+infra::Optional<hal::OutputPin> pinGreen;
```

Comment:
Is there a reason these two are not static within `main()`?

Reply:
Yes, but it's not a good reason. I'll update it.